rider-core/src/main/java/com/github/database/rider/core/dataset/builder/RiderDataSetBuilder.java

```diff
@@ -0,0 +1,27 @@
+package com.github.database.rider.core.dataset.builder;
+
+import org.dbunit.dataset.DataSetException;
+import org.dbunit.dataset.builder.DataSetBuilder;
+
+/**
+ * @author rmpestano
+ */
+public class RiderDataSetBuilder extends DataSetBuilder {
+
+    boolean uppercase;
+
+    public RiderDataSetBuilder() throws DataSetException {
+        super(false);
+        this.uppercase = false;
+    }
+
+    public RiderDataSetBuilder(boolean uppercase) throws DataSetException {
```

Comment:
Would it be better to have a builder method instead of the magic boolean constructor parameter?

Reply:
Agreed, also I think we can use [dbunit config](https://github.com/rmpestano/database-rider/blob/f964f8d36bc9b5e921101ee62f9df90bc663198c/rider-core/src/main/java/com/github/database/rider/core/api/configuration/DBUnit.java#L55) and as we are not going to inherit from DataSetBuilder anymore we can remove the parameter.